tests/PropTest.php

```diff
@@ -107,7 +107,7 @@ public function offsetUnset($offset) {
             array(null, 'foo', $obj),
             array('12345', 'bar', $obj),
             array(2012, 'year', $carbonInstance),
-            array(248, 'dayOfYear', $carbonInstance),
+            array(249, 'dayOfYear', $carbonInstance),
```

Comment:
How is this possible? Did Carbon get an update that made its day-of-year index one-based instead of zero-based?

Reply:
Yes. The second from below: https://carbon.nesbot.com/docs/#api-carbon-2

Reply:
I guess it makes sense when considering "day of the year".